sdks/python/apache_beam/typehints/trivial_inference_test.py

```diff
@@ -29,14 +28,12 @@
 global_int = 1
 
 
-@unittest.skipIf(sys.version_info >= (3, 6, 0) and
```

Comment:
We can try to reenable a few other tests, see: https://github.com/apache/beam/search?q=BEAM-6877&unscoped_q=BEAM-6877

Reply:
Since the remaining BEAM-6877 tests require some non-trivial changes (3 new opcodes), I'll leave that for another PR.